pkgs/by-name/fi/filius/package.nix

```
@@ -0,0 +1,126 @@
+{
+  stdenvNoCC,
+  lib,
+  fetchzip,
+  fetchurl,
+  makeWrapper,
+  makeDesktopItem,
+  copyDesktopItems,
+  jre,
+}:
+let
+  # we cache potentially unstable upstream inputs (icon and .zip file) via https://web.archive.org - this is a standard procedure in Nixpkgs
+  icon128 = fetchurl {
+    url = "https://web.archive.org/web/20240623123147/https://dl.flathub.org/repo/appstream/x86_64/icons/128x128/de.lernsoftware_filius.Filius.png";
+    sha256 = "sha256-xES6RWf2KiNsukD6GwTuPyiDI6SseADGLb1AkBSTgd8=";
```

```suggestion
    hash = "sha256-xES6RWf2KiNsukD6GwTuPyiDI6SseADGLb1AkBSTgd8=";
```
We should ask upstream to add this to the repository so all users get the increased image quality.

---

the only sources the flatpak has is the official filius download zip though.

---

i already asked about this: <https://github.com/NixOS/nixpkgs/pull/318005#discussion_r1654543818>